README.md

```diff
@@ -6,29 +6,12 @@ Goose is a database migration tool. Manage your database schema by creating incr
 
 ### Goals of this fork
 
-`github.com/pressly/goose` is a fork of `bitbucket.org/liamstask/goose` with the following changes:
-- No config files
-- [Default goose binary](./cmd/goose/main.go) can migrate SQL files only
-- Go migrations:
-    - We don't `go build` Go migrations functions on-the-fly
-      from within the goose binary
-    - Instead, we let you
-      [create your own custom goose binary](examples/go-migrations),
-      register your Go migration functions explicitly and run complex
-      migrations with your own `*sql.DB` connection
-    - Go migration functions let you run your code within
-      an SQL transaction, if you use the `*sql.Tx` argument
-- The goose pkg is decoupled from the binary:
-    - goose pkg doesn't register any SQL drivers anymore,
-      thus no driver `panic()` conflict within your codebase!
-    - goose pkg doesn't have any vendor dependencies anymore
-- We encourage using sequential versioning of migration files
-    (rather than timestamps-based versioning) to prevent version
-    mismatch and migration colissions
+`github.com/mc2soft/goose` is a fork of `github.com/pressly/goose` with the following changes:
+- Applying all missing migrations, regardless of latest migration version.
 
 # Install
 
-    $ go get -u github.com/pressly/goose/cmd/goose
+    $ go get -u github.com/mc2soft/goose/cmd/goose
```

Comment:
You don't expect us to merge this change `github.com/mc2soft/goose`, do you? :)

Can you iron this PR out and provide a high-level overview what is this PR supposed to solve? Thanks!

Reply:
This PR resolves #73. Ok, i will iron it to avoid "mc2soft" mentions)) But pls give me feedback, do you consider such issue?

Reply:
So in the high-level, this would find all the missing migrations -- ie. all migrations that are available but were not applied in the DB, right?

You can
1. list them
2. apply them

Reply:
Yes, with new commands:
1. status-missing
2. up-missing

Reply:
Now, do we need 1. at all? Can't `goose status` show these migrations as well by default?

Imho, 2. is explicit enough. I like it. It might even help people who insist on using Timestamp based migrations https://github.com/pressly/goose/issues/63.

Reply:
`status` show all migrations (could be a lot of them!), including missing ("pending"). But the new `status-missing` shows only missing migrations.

Reply:
I'd like someone from #63 to test this feature. I don't really have the same use case. Can you ping these guys, once you have the PR ready?